features/step_definitions/search_bar_steps.rb

```diff
@@ -0,0 +1,49 @@
+  Given('We have the following assignments exist:') do |table|
+    table.hashes.each do |assignment|
+      Assignment.create!(
+        assignment_name: assignment['assignment_name'],
+        repository_name: assignment['repository_name']
+      )
+    end
+  end
+
+  Given('I am on the assignments page') do
+    visit assignments_path
+  end
+
+  When('I enter {string} into the search bar') do |query|
```

Comment:
So is this

Reply:
Seems to be same as `'I fill in {string} with {string}'`

Reply:
That's right! Modified it accordingly.